### ISSUE TITLE
After adding a place, I want to see it immediately on the map

### ISSUE THREAD

**Issue by holgerd** [sozialhelden/wheelmap#464]
As a logged-in user who adds a new node using /nodes/new , I want to immediately see the node I created on the map.

Why:
We saw that some users create the same place multiple times. We want to prevent that in the future.

Alternative ideas on how to fix this:

- show a clear message after adding a node "Thanks for your node, it will be processed now and it will take some minutes to show up. You don't need to wait."
- Promise and send users confirmation email after the node they added is in the database

**Comment by schultyy** [comment]
We already show this message:
<img width="612" alt="screen shot 2016-12-19 at 11 29 18" src="https://cloud.githubusercontent.com/assets/235808/21309387/740ffb1c-c5de-11e6-8f44-a2540e5f438a.png">
We can definitely improve this message to point out better that it takes a bit until the node shows up on the map.

Concerning the email:
My feeling is, that this works fine if I create a new node occasionally but when I do this on a regular basis after the second email I already know that takes a bit. 
So does it make sense to introduce a setting where the user is able to turn off this email?

**Comment by holgerd** [comment]
@schultyy good point, the problem might just be that the notice is often overlooked.

A design challenge, maybe?

**Comment by Hoverbear** [comment]
I definitely overlooked the notice a couple times when adding things to wheelmap. :)

A dialog or "You did it!" page might be noticed more.
